components/feature/awesomebar/src/main/java/mozilla/components/feature/awesomebar/provider/HistoryMetadataSuggestionProvider.kt

```diff
@@ -70,6 +71,7 @@ class HistoryMetadataSuggestionProvider(
                 editSuggestion = result.key.url,
                 onSuggestionClicked = {
                     loadUrlUseCase.invoke(result.key.url)
+                    emitHistorySuggestionClickedFact()
```

Comment:
Looked into also testing this but going with the easy route of making `into()` static requires a bigger refactoring since now it's closely tied to `this`(HistoryMetadataSuggestionProvider), something that exceeds the scope of the ticket.